apps/src/codebridge/ControlButtons/index.tsx

```diff
@@ -130,6 +133,39 @@ const ControlButtons: React.FunctionComponent = () => {
     }
   };
 
+  const getRunTestTooltip = () => {
```

Comment:
If "run/test" are going to be referred to together a lot, I could see grouping these as "code actions" or something and using names like `getCodeActionsTooltip`, but no strong opinions.

Reply:
I'll rename it!